packages/plugin-react/src/index.ts

```diff
@@ -110,7 +118,10 @@ export default function viteReact(opts: Options = {}): PluginOption[] {
 
             plugins.push([
               await import('@babel/plugin-transform-react-jsx'),
-              { runtime: 'automatic' }
+              {
+                runtime: 'automatic',
+                importSource: opts.jsxImportSource ?? 'react'
```

Comment:
The `?? 'react'` part is unnecessary. Just for clarity?

Reply:
Yeah, it should be unnecessary. I will remove it.